gatsby-config.js

```diff
@@ -36,6 +36,13 @@ const plugins = [
   'gatsby-plugin-react-helmet',
   'gatsby-plugin-sitemap',
   'gatsby-plugin-twitter',
+  {
+    resolve: 'gatsby-theme-iterative-docs',
+    options: {
+      docComponent: require.resolve('./src/templates/doc.tsx'),
```

Comment:
Where is the `docComponent` being used in the theme? I'm not seeing it being used anywhere 🤔

Reply:
Whoops! I renamed this to `defaultTemplate` in `createPages` but didn't end up updating it here.